flexboxAPIspecV2.md

```diff
@@ -0,0 +1,663 @@
+
+
+
+Flexbox Layout for WinUI 
+===
+
+# Background
+
+
+
+
+Flexbox Layout is designed to provide an efficient way to layout, align, and distribute space among items within a container, specifically when the size of the device is either unknown or dynamic.
+
+Flexbox is a widely used layout schematic in web development, it allows developers to quickly create complex layouts with a standardized system. By introducing Flexbox to WinUI we can increase the layout choices developers have and cater to those who desire to use this popular layout in their WinUI applications.
+
+More on Flexbox available here: https://css-tricks.com/snippets/css/a-guide-to-flexbox/#aa-flexbox-properties
+
+
+# Remarks
+
+
+
+WinUI Offers several Layout options, such as StackLayout and Flowlayout. Usage of Flexbox Layout will behave similarly and interact by wrapping your desired elements in a Layout Control.
+
+#### Flex Containers vs Flex Items
+
+Flex Containers are elements that hold and specify the arangement of Flex Items. Containers have the following accepted properties:
```

Comment:
Still confused about what containers & items are. Is there a <FlexContainer> type? Or is the first child of the Flexbox a container and its children or items? Probably just needs a line or two of descriptive text and some example markup.